app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation-header.js

```diff
@@ -153,6 +160,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.$module.querySelector('.gem-c-layout-super-navigation-header__search-item-link')
       .setAttribute('hidden', 'hidden')
 
+    this.$superNavigationHeader = document.querySelector('.gem-c-layout-super-navigation-header')
```

Comment:
I wonder if it would make sense to move this variable assignment to line 100 of the file where we define the other top level variables (I think it's currently being assigned `null` there)? The variable could maybe also be called `$moduleParent` instead of `$superNavigationHeader` to be slightly clearer about what it is we're selecting and the relationship between the module and the super navigation header.

Then if we make `.gem-c-layout-super-navigation-header` a module level variable in this way, we'll be able to call it with `this.$superNavigationHeader` (or `this.$moduleParent`) inside the methods instead of passing it in the method parameters. We're not hiding this element in the DOM, only adding a class to it so it could be a bit confusing that it's being passed to `hide()` and `show()`. (We could even make toggling this class on the header into its own method that is called when show and hide are called, that could be the clearest thing to do in some ways.)